### PR TITLE
Improve dev tooling and add management commands

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

I suggest to use custom Django management commands to hold all our snippets we regularly run on the production system.
To get started, I added 4 of them to this PR, but I think it makes sense to search our ipython history for more snippets we should add there in future PRs. Also, we can add our OKR metrics here.

### Proposed changes

- Rename `dev-tools` directory to `tools`
- Refactor Django app structure
    - Add verbose names for all apps
    - Move general stuff into core app
- Add base classes for custom management commands
- Add dev tool to execute management commands
- Add management commands:
    - duplicate pages
    - initial SUMM.AI translation
    - find large files
    - find version inconsistencies (part of #1531)

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/core/management/commands/duplicate_pages.py [hunk 1]
```diff
@@ -86,6 +78,47 @@ def duplicate_pages(region, old_parent=None, new_parent=None):
         duplicate_pages(region, old_parent=old_page, new_parent=new_page)
 
 
-selected_region = Region.objects.first()
-logger.info("Duplicating pages for region %r", selected_region)
-duplicate_pages(selected_region)
+class Command(DebugCommand):
+    """
+    Management command du duplicate all pages of a region
```
Comment (JoeyStk) [inline integreat_cms/core/management/commands/duplicate_pages.py:83]:
```suggestion
    Management command du duplicate all pages of a region
```
I think this is a typo. Do you mean "to"?

Comment (timoludwig) [reply integreat_cms/core/management/commands/duplicate_pages.py:83]: Good catch!

File: integreat_cms/core/management/commands/find_missing_versions.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        return MODELS[model_str]
+    except KeyError as e:
+        raise ArgumentTypeError(
+            "Invalid model (must be either page, event or poi)"
+        ) from e
+
+
+class Command(LogCommand):
+    """
+    Management command to find large media files
```
Comment (JoeyStk) [inline integreat_cms/core/management/commands/find_missing_versions.py:35]:
```suggestion
    Management command to find missing versions
```

**Review by JoeyStk — COMMENTED** [review]
A positive, general observation that I made: ./dev-tools/run.sh was notoriously slow on my PC. I'm not sure which change is responsible for it, but ./tools/run.sh is A LOT faster!

File: integreat_cms/core/management/commands/find_missing_versions.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        return MODELS[model_str]
+    except KeyError as e:
+        raise ArgumentTypeError(
+            "Invalid model (must be either page, event or poi)"
+        ) from e
+
+
+class Command(LogCommand):
+    """
+    Management command to missing versions
```
Comment (JoeyStk) [inline integreat_cms/core/management/commands/find_missing_versions.py:35]: I'm sorry if I'm a bit picky here, but I think there's still a verb missing here :sweat_smile: I think you mean "find"?

Comment (timoludwig) [reply integreat_cms/core/management/commands/find_missing_versions.py]: Haha thanks, yes that's the right amount of picky-ness :laughing:

**Review by JoeyStk — APPROVED** [review]
Thank you for this very cool PR!
Every tool worked very well for me, so now I don't have any comments anymore :)

**Review by MizukiTemma — APPROVED** [review]
Thank you 🚀

**Pull request merged**